content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+
+##### 4. Create an account
+
+I choose to use the Object Store Web GUI to create an account.
+
+Refer to this document - [Using the Object Store Interface](https://docs.datafabric.hpe.com/72/MapROverview/create-account.html#create_account__section_fsm_1hn_nrb).
+
+Create an account using Object Store Web GUI - 1
+<a href="https://ibb.co/SyW9MKK"><img src="https://i.ibb.co/XLNvKzz/Object-Store-Create-Account-1.png" alt="Object-Store-Create-Account-1" border="0"></a>
+
+Enter the following in "Default Bucket Place":
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+      {
+        "Sid": "GrantAdminPutPermissions",
+        "Effect": "Allow",
+        "Principal": "arn:primary:default:user:mapr",
+        "Action":"s3:PutObject",
+        "Resource":"arn:aws:s3:::${bucket}/*"
+      },
+      {
+        "Sid":"GrantAnonymousReadPermissions",
+        "Effect":"Allow",
+        "Principal": "*",
+        "Action":["s3:GetObject"],
+        "Resource":["arn:aws:s3:::${bucket}/*"]
+      }
+  ]
+}
+```
+
+HPE Ezmeral Data Fabric Object Store is an on-premises object storage service compatible with Minio.
+Some concepts such as Domain and Default Account do not exist in public cloud object storage services such as AWS S3.
+But the policy for Bucket and IAM User is compatible with the policy in public cloud object storage.
+For the Bucket Policy here, you can refer to AWS S3 [Bucket policy examples](https://docs.aws.amazon.com/AmazonS3/latest/userguide/example-bucket-policies.html) and HPE Ezmeral Data Fabric Object Store document - [Access Policies](https://docs.datafabric.hpe.com/72/MapROverview/object-store-policies.html).
+
+After creating the user account, you can use the below command to view it:
+
+```shell
+sudo -u mapr /opt/mapr/bin/mc admin account list {ADMIN_ALIAS} domain=primary --json
+```
+
+Sample output:
+
+```json
+{
+ "name": "default",
+ "id": 0,
+ "admin": "mapr",
+ "labelname": "default",
+ "minrepl": 2,
+ "desiredrepl": 3,
+ "usercount": 2
+}
+{
+  // # 👇 The account you just created.
+ "name": "s3test",
+ "id": 1,
+ "admin": "mapr",
+ "def_bucket_policy": {
+  "Version": "2012-10-17",
+  "Statement": [
+   {
+    "Sid": "GrantAdminPutPermissions",
+    "Effect": "Allow",
+    "Principal": {
+     "AWS": [
+      "arn:primary:default:user:mapr"
+     ]
+    },
+    "Action": [
+     "s3:PutObject"
+    ],
+    "Resource": [
+     "arn:aws:s3:::${bucket}/*"
+    ]
+   },
+   {
+    "Sid": "GrantAnonymousReadPermissions",
+    "Effect": "Allow",
+    "Principal": {
+     "AWS": [
+      "*"
+     ]
+    },
+    "Action": [
+     "s3:GetObject"
+    ],
+    "Resource": [
+     "arn:aws:s3:::${bucket}/*"
+    ]
+   }
+  ]
+ },
+ "size": 22871,
+ "labelname": "default",
+ "topology": "/data/default-rack",
+ "minrepl": 1,
+ "desiredrepl": 1,
+ "usercount": 1,
+ "bucketcount": 1
+}
+```
+
+#### Create an IAM User in the non-default Account you created just now
```

Comment:
Recommend rephrasing to "Create an IAM User in the non-default account just created"